### PR TITLE
When constant change, please add update of constant

### PR DESCRIPTION
**Pull request by defrance**

### CODE REVIEW

File: htdocs/install/mysql/migration/4.0.0-5.0.0.sql [hunk 1]
```diff
@@ -24,6 +24,8 @@
 -- -- VPGSQL8.2 DELETE FROM llx_usergroup_user      WHERE fk_user      NOT IN (SELECT rowid from llx_user);
 -- -- VMYSQL4.1 DELETE FROM llx_usergroup_user      WHERE fk_usergroup NOT IN (SELECT rowid from llx_usergroup);
 
+-- after changing const name, please insure that old constant was rename
+UPDATE llx_const SET name = 'THIRDPARTY_DEFAULT_CREATE_CONTACT' WHERE name='MAIN_THIRPARTY_CREATION_INDIVIDUAL'
```
Comment (philippe-opendsi) [inline htdocs/install/mysql/migration/4.0.0-5.0.0.sql]: hello, you update from 
MAIN_THIRPARTY_CREATION_INDIVIDUAL
is it not
MAIN_THIR**D**PARTY_CREATION_INDIVIDUAL
with D ?

Comment (defrance) [reply htdocs/install/mysql/migration/4.0.0-5.0.0.sql]: your partly right, MAIN_THIRPARTY_CREATION_INDIVIDUAL was changed to MAIN_THIRDPARTY_CREATION_INDIVIDUAL in 3.9 ... so we need 2 update...

Comment (atm-maxime) [reply htdocs/install/mysql/migration/4.0.0-5.0.0.sql]: Don't forget semicolons at the end of lines... I'll push a fix

Comment (defrance) [reply htdocs/install/mysql/migration/4.0.0-5.0.0.sql]: Soooorry, Thanks Max, i'm not good this time

**Pull request merged**